plugin/inc/views/view.class.php

```diff
@@ -1445,14 +1445,41 @@ protected function evalIfFunction($string, $global_variables = []): string
         return $string;
     }
 
-    public static function modifyTitleBasedOnUnpaidInvoices(array $rowData, string &$title, string &$color): void
+    public static function getPartnerTitleBasedOnUnpaidInvoices(int $numberOfUnpaidInvoices, $valueOfUnpaidInvocies, bool $withLineBreaks = false): string
```

Comment:
My initial idea was to leave out the line breaks from the function, but since you introduced them parametrized, it would be a more flexible approach to take in the number of line breaks that should percede the title:
```suggestion
    public static function getPartnerTitleBasedOnUnpaidInvoices(int $numberOfUnpaidInvoices, $valueOfUnpaidInvocies, int $lineBreaksBefore = 0): string
```
Then later use a loop to add as many line breaks as necessary.